packages/ui-react/src/containers/ShelfList/ShelfList.tsx

```diff
@@ -70,29 +69,27 @@ const ShelfList = ({ rows }: Props) => {
           const visibleTilesDelta = parseTilesDelta(posterAspect);
 
           return (
-            <div
+            <section
               key={`${index}_${playlist.id}`}
-              role="row"
               className={classNames(styles.shelfContainer, { [styles.featured]: featured })}
               data-testid={testId(`shelf-${featured ? 'featured' : type === 'playlist' ? slugify(title || playlist?.title) : type}`)}
+              aria-label={title || playlist?.title}
```

Comment:
I have chosen to keep it simple and apply a `aria-label `instead of `aria-labelledby`. If I chose the latter I need to create a unique ID and pass it the component. In all cases we use the hook `useOpaqueId` for that. But I cannot use that hook inside a loop.

I find it cumbersome to pass the generated ID to the component. Potentially using the `slugify` utility for it.